### PR TITLE
Credentials example

### PR DESCRIPTION
**Pull request by nikgraf**

### CODE REVIEW

File: aws-node-env-variables-encrypted-in-a-file/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+This example demonstrates how to store secrets like API keys encrypted in your repository while providing them as environment variables to your AWS Lambda functions.
+
+## Use-cases
+
+- Provide secrets like API keys to your Lambda functions
+
+## Why?
+
+While repository hosting services like Github or Bitbucket have very high security standards it's recommended to not store your unencrypted secrets there. In addition in larger teams not everybody needs to have access to theses secrets of your production environment.
```
Comment (mthenw) [inline aws-node-env-variables-encrypted-in-a-file/README.md]: theses -> those?

File: aws-node-env-variables-encrypted-in-a-file/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "env-variables-encrypted-in-a-file",
+  "version": "1.0.0",
+  "description": "Serverless example managing secrets in an encrypted file",
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "serverless-secrets-plugin": "^0.0.1"
```
Comment (mthenw) [inline aws-node-env-variables-encrypted-in-a-file/package.json]: hmm, weird, I can't find it on npm

**Comment by mthenw** [comment]
I can't find this plugin so I'm not sure how this thing works. 

In general I think we should use https://aws.amazon.com/kms/ somehow.

File: aws-node-env-variables-encrypted-in-a-file/package.json (hunk 2 above)
Comment (nikgraf) [inline aws-node-env-variables-encrypted-in-a-file/package.json]: will publish it soon. want to add a readme first …

**Comment by nikgraf** [comment]
@mthenw why is it not secure at all?

**Comment by nikgraf** [comment]
In the AWS docs it says: `When you create or update Lambda functions that use environment variables, AWS Lambda encrypts them using the AWS Key Management Service.`

But they also mention

```
Storing Sensitive Information

For sensitive information, such as database passwords, we recommend you use client-side encryption using AWS Key Management Service and store the resulting values as Ciphertext in your environment variable. You will need to include logic in your Lambda function code to decrypt these values.
```

So in the end the Lambda function needs a password/key to decrypt the Ciphertext. How do I safely provide the Lambda function with this password/key? How is this safer than just passing in the environment variable as plaintext and then Lambda storing it in KMS?

**Comment by nikgraf** [comment]
@mthenw I left an important note on top of the readme referring to the AWS docs. I asked the question on Twitter and in the Serverless forum slack. Once we have a good answer I will extend this example.

**Comment by nikgraf** [comment]
@mthenw anything to add here? Ryan clarified a lot here: https://gist.github.com/nikgraf/ef25181cf5f2bf85e9dd94184ec5446a#gistcomment-1931308

**Pull request merged**